### PR TITLE
Update owners files for autoscaling

### LINKED ISSUE #1111: Create an OWNERS file
ref: https://github.com/kubernetes/community/issues/1721

This repo is listed in sigs.yaml as belonging to sig-autoscaling, but it lacks an OWNERS file.  Can you please add one with the appropriate list of approvers and reviewers? eg:
- [robust example: approvers, reviewers, aliases](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/OWNERS)
- [simple example: only approvers, no aliases](https://github.com/kubernetes-incubator/external-storage/blob/master/OWNERS)

For more information see the [OWNERS docs](https://go.k8s.io/owners)

/sig autoscaling

/assign @mwielgus @DirectXMan12 
sig-autoscaling chairs

/assign @aleksandra-malinowska @bskiba @MaciekPytel 
as folks who've merged PR's here recently

### PR DESCRIPTION
**Pull request by bskiba**
fixes #1111

### CODE REVIEW

**Comment by bskiba** [comment]
@mwielgus @MaciekPytel

File: cluster-autoscaler/OWNERS [hunk 1]
```diff
@@ -1,9 +1,11 @@
 approvers:
-- fgrzadkowski
 - mwielgus
```
Comment (MaciekPytel) [inline cluster-autoscaler/OWNERS]: mwielgus is already a top level owner, so probably redundant here (or I want to be included as well :) )

**Comment by aleksandra-malinowska** [comment]
Do we want to add an OWNERS file for builder/ as well? It's specific to Cluster Autoscaler and changes there are necessary e.g. when adding extra build dependency (that won't be included in the release image anyway). However, with the new setup it will require approval by top-level OWNERS. Given that changes such as adding dependencies to be included in the release image only require cluster-autoscaler/OWNERS approval, this doesn't seem right.

**Comment by bskiba** [comment]
@aleksandra-malinowska shouldn't the builder be used for all autoscaling images (I have some wip that adds build-in-docker to vpa)? Then this would imply that top-level OWNERS is right for it.

**Comment by aleksandra-malinowska** [comment]
If this is going to be the case, then sure, it should probably require top-level approval, to avoid one project's dependencies' update from breaking another one. Although I don't think it's the best idea to share it, given that each project may require different dependencies. In particular, sharing it would mean we'd always have to sync upgrading to use newer version of Go for all projects (which would in some cases mean synchronized godeps update for all of them.)

**Comment by MaciekPytel** [comment]
Maybe we should add addon-resizer owners too while we're doing it? I guess that would be @x13n and @kawych?

**Comment by aleksandra-malinowska** [comment]
addon-resizer/OWNERS already exists, but +1 to updating it (AFAIU Q-Lee is no longer working on this, and current maintainers are missing)

File: cluster-autoscaler/OWNERS (hunk 1 above)
Comment (bskiba) [reply cluster-autoscaler/OWNERS]: Done

**Comment by bskiba** [comment]
Added addon-resizer changes.

**Comment by bskiba** [comment]
@kawych @x13n FYI

**Pull request merged**